src/Operations/ConvertContoursToMeshes.cc

```diff
@@ -396,6 +396,9 @@ bool ConvertContoursToMeshes(Drover &DICOM_data,
                 std::list<pairing_t> pairs;
 
                 const auto set_union_is_empty = [](const std::set<size_t> &A, const std::set<size_t> &B) -> bool {
+                    //if both sets are empty their union is not empty
+                    if (A.empty() && B.empty()){return false;}
```

Comment:
don't need curly brackets for inline if statement
```suggestion
                    if (A.empty() && B.empty()) return false;
```